docs/launch-profiles.md

```diff
@@ -0,0 +1,65 @@
+﻿# Launch Profiles
+
+The _Launch Profiles_ feature of the .NET Project System allows developers to specify multiple ways of running their project, each with a different set of options, and quickly switch between them in Visual Studio. For example, when developing a web application you may have one profile that runs your project under the Kestrel web server, and another that launches it under IIS Express. Or, when creating a console application, different profiles may specify different command line arguments.
+
+## Example
+
+The launch profiles for a particular project (if any) are stored in the Properties\launchSettings.json file. Below is an example based on a launchSettings.json found in this repo. Note that within Visual Studio it is more common to define and update launch profiles through the UI (via the project properties in VS2019 and earlier, or the Launch Profiles UI in VS2022 and later) but the file can be edited directly.
+
+``` json
+{
+  "profiles": {
+    "Start (with native debugging)": {
+      "commandName": "Executable",
+      "executablePath": "$(DevEnvDir)devenv.exe",
+      "commandLineArgs": "/rootsuffix $(VSSDKTargetPlatformRegRootSuffix) /log",
+      "environmentVariables": {
+        "VisualBasicDesignTimeTargetsPath": "$(VisualStudioXamlRulesDir)Microsoft.VisualBasic.DesignTime.targets",
+        "FSharpDesignTimeTargetsPath": "$(VisualStudioXamlRulesDir)Microsoft.FSharp.DesignTime.targets",
+        "CSharpDesignTimeTargetsPath": "$(VisualStudioXamlRulesDir)Microsoft.CSharp.DesignTime.targets",
+        "CPS_DiagnosticRuntime": "1",
+        "CPS_MetricsCollection": "1"
+      },
+      "nativeDebugging": true
+    }
+  }
+}
+```
+
+This file contains a single launch profile named "Start (with native debugging)" containing various settings:
+
+- `commandName`: This is the most important setting, and the only required one. We'll come back to it in a moment.
+- `executablePath`: The path to an executable (.exe) to run when this launch profile is selected, and the project is "started". In this starting the project will run devenv.exe.
+- `commandLineArgs`: The arguments to supply to the specified executable.
+- `environmentVariables`: A collection of name/value pairs, each specifying an environment variable and value to set. A setting in a profile can contain any arbitrary data so long as it can be represented in JSON (and even that isn't always a requirement; see "In-memory settings and profiles", below).
+- `nativeDebugging`: Specifies that the native code debugging engine should be used in addition to the managed code debugging engine.
+
```

Comment:
To avoid someone who is skim reading, or who stopped reading here, taking this as a reference list of settings:

```suggestion

This is not an exhaustive list of settings, as we will discuss below.

```